### PR TITLE
Add Translate Support

### PR DESCRIPTION
**Pull request by Varorbc**
Add Chinese Support

### CODE REVIEW

**Comment by Varorbc** [comment]
#155 #366

File: src/app.config.js [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
   angular.module('app.config', [])
     .constant('BASE_URL', 'http://localhost:50000')
+    .constant('LANGUAGE','zh-cn')
```
Comment (niemyjski) [inline src/app.config.js]: Is there any chance we can ditch the language constants. I think angular translate will do this automatically (https://angular-translate.github.io/docs/#/guide/09_language-negotiation) via: `$translateProvider
  .translations('en', { /* ... */ })
  .translations('de', { /* ... */ })
  .determinePreferredLanguage();`

File: src/app.js [hunk 2]
```diff
@@ -64,7 +65,15 @@
     'app.stack',
     'app.status'
   ])
-  .config(function ($compileProvider, $httpProvider, $locationProvider, $stateProvider, $uiViewScrollProvider, $urlRouterProvider, dialogsProvider, gravatarServiceProvider, RestangularProvider, BASE_URL, EXCEPTIONLESS_API_KEY, $ExceptionlessClient, stripeProvider, STRIPE_PUBLISHABLE_KEY, USE_HTML5_MODE) {
+  .config(function ($compileProvider, $httpProvider, $locationProvider, $stateProvider, $uiViewScrollProvider, $urlRouterProvider, $translateProvider, dialogsProvider, gravatarServiceProvider, RestangularProvider, BASE_URL, EXCEPTIONLESS_API_KEY, $ExceptionlessClient, stripeProvider, STRIPE_PUBLISHABLE_KEY, USE_HTML5_MODE, LANGUAGE) {
+    console.log(LANGUAGE);
+    var lang = window.localStorage.lang ||LANGUAGE|| 'zh-cn';
+    $translateProvider.preferredLanguage(lang);
```
Comment (niemyjski) [inline src/app.js]: Is there any chance we can ditch the language constants. I think angular translate will do this automatically (https://angular-translate.github.io/docs/#/guide/09_language-negotiation) via: `$translateProvider
  .translations('en', { /* ... */ })
  .translations('de', { /* ... */ })
  .determinePreferredLanguage();`

**Comment by niemyjski** [comment]
Looks good, how has it worked out for you? Have you had a chance to run it? I think we can just use angular to detect the specified language automatically (https://angular-translate.github.io/docs/#/guide/09_language-negotiation) via: `$translateProvider
  .translations('en', { /* ... */ })
  .translations('de', { /* ... */ })
  .determinePreferredLanguage();` Have you tried this.

**Comment by niemyjski** [comment]
It looks like there are a few merge conflicts, please let me know if you need any assistance with this.

**Comment by niemyjski** [comment]
@Varorbc looks good, is there anything else we need or is this ready for more testing / merging into feature branch for additional qa?

**Comment by Varorbc** [comment]
yes,you can test and merge pr

**Comment by niemyjski** [comment]
Thanks for the PR! I'll try and go over it this week and do some testing on it :)

**Comment by niemyjski** [comment]
Sorry I've taken so long to get to this, will try and get this merged in real soon.

**Pull request merged**